### ISSUE TITLE
Update recipes reported as outdated on Repology

### ISSUE THREAD

**Issue by Begasus** [haikuports/haikuports#6235]
See the list at [Repology](https://repology.org/projects/?inrepo=haikuports_master&outdated=1)

**Comment by korli** [comment]
Good idea, but I would avoid not widely up-to-date  software: https://repology.org/projects/?search=&maintainer=&category=&inrepo=haikuports_master&notinrepo=&repos=&families=15-&repos_newest=&families_newest=10-&outdated=on
